src/main/java/io/cdap/e2e/utils/CdfHelper.java

```diff
@@ -100,12 +100,12 @@ default void succeedPipelineVerification() {
 
   default int recordIn() {
     String incount = CdfPipelineRunLocators.recordsInCount.getText();
-    return Integer.parseInt(incount);
+    return Integer.valueOf(incount.replaceAll(",", "").toString());
```

Comment:
why do we need to make this change?

Reply:
we are removing comma from the count value

Reply:
don't need to change valueOf->parseInt.